src/java.base/unix/classes/sun/security/provider/NativePRNG.java

```diff
@@ -203,11 +203,14 @@ static boolean isAvailable() {
     }
 
     // constructor, called by the JCA framework
-    public NativePRNG() {
+    public NativePRNG(SecureRandomParameters params) {
         super();
```

Comment:
nit: maybe super() not needed?

Reply:
It was already there before, but yes it doesn't do anything useful. The other impls I've checked don't use it, so I'll remove.